components/KidProfile/KidProfile.styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export const StyledKidProfile = styled.div`
+  align-items: center;
+  column-gap: 1.6rem;
+  padding: 1.2rem;
+  text-align: center;
+  transition: 0.5s;
+  align-items: center;
+
+  img {
+    width: 200px;
+   height: 200px;
+  }
+`;
+
+export const StyledGoBackButton = styled.button`
+  background-color: #f0f0f0;
+  font-size: 16px;
+  color: #333;
+  border: none;
+  border-radius: 4px;
+  padding: 8px 16px;
+  cursor: pointer;
+  &:hover {
+    background-color: #6ffffa;
+`;
```

Comment:
If you use this styled button component in the future too for other components you can think about working with variants. [See this example for future considerations.](https://github.com/mbosselmann/tourio/blob/main/components/StyledButton.js)

Reply:
Thanks! I will use this in my next US.